src/Navigation/View.elm

```diff
@@ -1,30 +1,67 @@
 module Navigation.View exposing (..)
 
 
-import Html exposing (div, text, a)
+import Html exposing (div, text, a, span)
 import Html.Events exposing (onClick)
 import Html.Attributes exposing (style, href, id)
+import Html.CssHelpers
+
+
+import Navigation.NavigationCss as NavigationCss
 
 
 import Navigation.Messages exposing (..)
 
 
-menuView : Html.Html Msg
-menuView =
-    div [ style [("background-color", "#000000"), ("color", "#FFFFFF")] ]
+import Routing.Models exposing (..)
+
+
+{ class } =
+  Html.CssHelpers.withNamespace "navigation"
+
+menuView : Route ->  Html.Html Msg
+menuView route =
+  let
+    
+    isPostsSelected =
+      route == PostsRoute
+      
+    isAdminSelected =
+      route == AdminRoute
+       
+  in
+    div [ class [NavigationCss.Container] ]
         [ div []
-              [ menuLink ShowPosts "btnPosts" "Posts"
-              , text " | "
-              , menuLink ShowAdmin "btnAdmin" "Admin"
+              [ menuLink ShowPosts "btnPosts" "Posts" isPostsSelected
+              , menuLink ShowAdmin "btnAdmin" "Admin" isAdminSelected
               ]
         ]
 
 
-menuLink : Msg -> String -> String -> Html.Html Msg
-menuLink message viewId label =
-    a [ id viewId
-      , href "javascript://"
-      , onClick message
-      , style [("text-decoration", "none"), ("color", "#FFFFFF")]
-      ]
-      [ text label ]
\ No newline at end of file
+menuLink : Msg -> String -> String -> Bool -> Html.Html Msg
+menuLink message viewId label isSelected =
+  let
+    
+    menuItem =
+      case isSelected of
+        True -> 
+          span [ id viewId
+               , class'
+               ]
+               [ text label ]
+        _ ->
+          a [ id viewId
+            , href "javascript://"
+            , onClick message
+            , class'
+            ]
+            [ text label ]
+
+
+    class' =
```

Comment:
We're repeating the same condition to a realy small chunk of code. Can't we move the class' logic up? Like this"
```Elm
case isSelected of
  True -> 
    span [ id viewId
         , class [NavigationCss.NavItem, NavigationCss.NavItemSelected]
         ]
         [ text label ]
  _ ->
    a [ id viewId
      , href "javascript://"
      , onClick message
      , class[NavigationCss.NavItem]
      ]
      [ text label ]
```